pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -1097,6 +1106,534 @@ ASTSqlStatement SqlStatement(String initiator, String terminator)  :
 	}
 }
 
+/**
+ * @see https://docs.oracle.com/en/database/oracle/oracle-database/18/lnpls/SELECT-INTO-statement.html
+ * @see https://docs.oracle.com/en/database/oracle/oracle-database/18/lnpls/img_text/select_into_statement.html
+ */
+ASTSelectIntoStatement SelectIntoStatement() :
+{}
+{
+    <SELECT> [ <DISTINCT> { jjtThis.setDistinct(true); }
+               | <UNIQUE> { jjtThis.setUnique(true); }
+               | <ALL> { jjtThis.setAll(true); } ] 
+
+    SelectList()
+    ( IntoClause() | BulkCollectIntoClause() )
+    RestOfStatement()
+
+    { return jjtThis; }
```

Comment:
Yes, I've seen this. I didn't change it for and just added it for the new nodes, too.

You can see one usage here:

https://github.com/pmd/pmd/blob/2416b88725ad89d0e31dd6f214a8955bc972405a/pmd-plsql/etc/grammar/PldocAST.jjt#L607-L611

There are many places, where the child node is captured and the image string is built which produces again a string representation of the current AST part...
I guess, it's more like a convenience for developing/debugging, since you can look at the "image" of a node to see, what this is, without the need to know the AST structure/node names.